app/router.js

```diff
@@ -58,6 +58,16 @@ Router.map(function() {
   this.route('school', { path: 'schools/:school_id'});
   this.route('assign-students', {path: '/admin/assignstudents'});
   this.route('myprofile');
+  this.route('course-rollover');
+  this.route('curriculumInventoryReports', {
```

Comment:
ok, so this won't prevent ppl from visiting that path.
i had some logic in beforeModel that would check perms, however that did not jive with authn if you went directly to a ci route (e.g. if you have it bookmarked).